src/main/java/org/gridsuite/modification/server/dto/VoltageInitBusModificationInfos.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * @author Ayoub LABIDI <ayoub.labidi at rte-france.com>
+ */
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Schema(description = "Voltage init generator modfication infos")
+public class VoltageInitBusModificationInfos {
+    @Schema(description = "Bus id")
+    private String busId;
+
+    @Schema(description = "Voltage set point")
+    private Double v;
+
```

Comment:
We must also modify the voltage angle of the bus, in addition to his voltage magnitude (see remark in PR : https://github.com/gridsuite/voltage-init-server/pull/57)